virtualizarr/xarray.py

```diff
@@ -94,6 +102,18 @@ def open_virtual_dataset(
     if common:
         raise ValueError(f"Cannot both load and drop variables {common}")
 
+    if cftime_variables is None:
+        cftime_variables = []
+    elif isinstance(cftime_variables, str):
+        cftime_variables = [cftime_variables]
+    else:
+        cftime_variables = list(cftime_variables)
+
+    if not set(cftime_variables).issubset(set(loadable_variables)):
+        raise ValueError(
+            "All ``cftime_variables`` must be included in ``loadable_variables``"
+        )
```

Comment:
Nit: We can make this error message a little more useful by displaying the set of `cftime_variables` which were not in `loadable_variables`.